jcli/src/jcli_app/key.rs

```diff
@@ -99,6 +110,40 @@ pub struct ToPublic {
     output_file: OutputFile,
 }
 
+#[derive(StructOpt, Debug)]
+pub struct Sign {
+    /// path to file with bech32-encoded secret key
+    ///
+    /// supported key formats are: ed25519, ed25519bip32, ed25519extended and sumed25519_12
+    #[structopt(long = "secret-key", short = "s")]
+    secret_key: PathBuf,
+
+    /// path to file with data to sign, if no value is passed, standard input will be used
+    #[structopt(long = "data", short = "d")]
+    data: Option<PathBuf>,
+
+    /// path to file to write signature into, if no value is passed, standard output will be used
+    #[structopt(long = "output", short = "o")]
+    output: Option<PathBuf>,
+}
+
+#[derive(StructOpt, Debug)]
+pub struct Verify {
+    /// path to file with bech32-encoded public key
+    ///
+    /// supported key formats are: ed25519, ed25519bip32 and sumed25519_12
+    #[structopt(long = "public-key", short = "p")]
+    public_key: PathBuf,
+
+    /// path to file with signed data, if no value is passed, standard input will be used
+    #[structopt(long = "data", short = "d")]
+    data: Option<PathBuf>,
+
+    /// path to file with signature
+    #[structopt(long = "signature", short = "s")]
```

Comment:
the short here is confusing with the short for the `sign` command. better no shorts than wearing the same one as everyone else